src/components/dataset/overview/dataset-properties.vue

```diff
@@ -0,0 +1,93 @@
+<!--
+Copyright 2023 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+
+<template>
+  <table v-if="properties.length > 0" id="dataset-properties" class="table">
+    <thead>
+      <tr>
+        <th>{{ $t('header.name') }}</th>
+        <th>{{ $t('header.updatedBy') }}</th>
+      </tr>
+    </thead>
+    <tbody>
+      <template v-for="(property) in properties" :key="property.name">
+        <tr>
+          <!-- we have to show property name even if there is no associated form -->
+          <td :rowspan="property.forms.length || 1">
+            {{ property.name }}
+          </td>
+          <td>
+            <router-link v-if="property.forms.length > 0" :to="publishedFormPath(projectId, property.forms[0].xmlFormId)" :title="property.forms[0].name">
```

Comment:
Could you add assertions about the links in this component, of the `to` prop and the `title` attribute? And add another test that the `to` prop is correct if the user is a project viewer?